lib/checks/color/color-contrast-evaluate.js

```diff
@@ -188,4 +148,64 @@ function colorContrastEvaluate(node, options, virtualNode) {
   return isValid;
 }
 
-export default colorContrastEvaluate;
+function findPseudoElement(vNode, {
+  pseudoSizeThreshold = 0.25,
+  ignorePseudo = false
+}) {
+  if (ignorePseudo) {
+    return;
+  }
+  const rect = vNode.boundingClientRect;
+  const minimumSize = rect.width * rect.height * pseudoSizeThreshold;
+  do {
```

Comment:
This loop is going to be slow. Before we memozied this part of the loop so looking up the parent chain wasn't so bad for every node as shared ancestors would only run through it once and then get cached. However the memoization has been removed so each element will be run through this check for every descendant.